python-avocado.spec

```diff
@@ -102,7 +102,7 @@ these days a framework) to perform automated testing.
 
 %build
 %if 0%{?rhel}
-sed -e 's/"PyYAML>=4.2b2"/"PyYAML>=3.12"/' -i optional_plugins/varianter_yaml_to_mux/setup.py
+sed -e 's/"PyYAML>=4.2b2"/"PyYAML>=3.13"/' -i optional_plugins/varianter_yaml_to_mux/setup.py
```

Comment:
this should be kept to 3.12 to ensure compatibility with python 3.12. see https://github.com/richtja/avocado/commit/8be7c4fb325aa875e7d752975d61cd89f9239e7b